### PR TITLE
standarize make test target

### PR DESCRIPTION
**Pull request by emzubair**
## Standardize the make test targets
Added target for test & clean

**Packages added**

- pytest-cov

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @emzubair! I've created [OSPR-4828](https://openedx.atlassian.net/browse/OSPR-4828) to keep track of it in JIRA, where we prioritize reviews.  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "@edx/edx-proctoring",
-  "//": "Be sure to update the version number in edx_proctoring/__init__.py",
```
Comment (feanil) [inline package.json:3]: Why did you remove this line?

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .PHONY: help upgrade requirements clean quality requirements docs \
 	test test-all coverage pii_check \
 	compile_translations dummy_translations extract_translations \
-	fake_translations pull_translations push_translations
+	fake_translations pull_translations push_translations test
```
Comment (feanil) [inline Makefile:4]: Add the other quality and test related targets to `PHONY` as well.

**Comment by emzubair** [comment]
@feanil  updated the `PR`

**Comment by edx-webhook** [comment]
@emzubair 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-proctoring&entry.867055334=https://github.com/edx/edx-proctoring/pull/697&entry.1484655318=https://github.com/emzubair&entry.752974735=2020-07-16+13:00&entry.1917517419=2020-07-21+19:58&entry.2133058324=Yes) so we can improve your experience in the future.
